examples/grid2grid/evaUtilities.py

```diff
@@ -0,0 +1,132 @@
+import matplotlib.pyplot as plt
+from config.config_manager import load_use_case_config
+import numpy as np
+from mpl_toolkits.mplot3d import proj3d
+from matplotlib.patches import Rectangle, Circle, FancyBboxPatch, FancyArrowPatch
+import mpl_toolkits.mplot3d.art3d as art3d
+import mpl_toolkits.mplot3d as mp3d
+import matplotlib.patches as mpatches
+
+
+class Arrow3D(FancyArrowPatch):
+    def __init__(self, xs, ys, zs, *args, **kwargs):
+        FancyArrowPatch.__init__(self, (0,0), (0,0), *args, **kwargs)
+        self._verts3d = xs, ys, zs
+
+    def draw(self, renderer):
+        xs3d, ys3d, zs3d = self._verts3d
+        xs, ys, zs = proj3d.proj_transform(xs3d, ys3d, zs3d, renderer.M)
+        self.set_positions((xs[0], ys[0]), (xs[1], ys[1]))
+        FancyArrowPatch.draw(self, renderer)
+
+
+def wrap_to_pi(angle):

```

Comment:
doesn't seem like this is used anywhere?

Reply:
removed